cmd/mayactl/app/command/stats_helper.go

```diff
@@ -32,16 +34,10 @@ const (
 	timeout = 5 * time.Second
 )
 
-// getVolDetails gets response in json format of a volume from m-apiserver
+// GetVolDetails gets response in json format of a volume from m-apiserver
 func GetVolDetails(volName string, namespace string, obj interface{}) error {
-	addr := os.Getenv("MAPI_ADDR")
-	if addr == "" {
-		err := util.MAPIADDRNotSet
-		fmt.Printf("error getting env variable: %v", err)
-		return err
-	}
 
-	url := addr + "/latest/volumes/info/" + volName
+	url := mapiserver.GetURL() + "/latest/volumes/info/" + volName
```

Comment:
Please give some comments why this utility function is needed and move "/latest/volumes/info" to constant var "InfoAPIEndpoint".

Reply:
ok